guides/common/modules/proc_generating-host-monitoring-reports.adoc

```diff
@@ -36,22 +37,22 @@ If you want to generate the report as a background task, you can use the `hammer
 +
 [NOTE]
 ====
-If you want to generate a subscription entitlement report, you have to use the `Days from Now` option to specify the latest expiration time of entitlement subscriptions.
-You can use the `no limit` value to show all entitlements.
+If you want to generate a subscription general report, you have to use the `Days from Now` option to specify the latest expiration time of general subscriptions.
+You can use the `no limit` value to show all inputs.
```

Comment:
Subscriptions, not inputs.  (Yes, subscriptions are still around in your manifest, and that's what this report shows :)
```suggestion
You can use the `no limit` value to show all subscriptions.
```